offices/views.py

```diff
@@ -1,3 +1,60 @@
-from django.shortcuts import render
+from django.views        import View
+from django.http         import JsonResponse
+from django.utils        import timezone
 
-# Create your views here.
+from offices.models      import Building
+from reservations.models import Reservation
+
+
+class OfficeView(View):
```

Comment:
Building을 기준으로 데이터를 다루기 때문에 BuildingView가 더 적합해 보이네요.
```suggestion
class BuildingView(View):
```